### PR TITLE
Frontend

### PR DESCRIPTION
**Pull request by caseynunes**

### CODE REVIEW

File: xconnect_frontend/xconnect/public/st-xaviers-college.jpg
Comment (sea-rod) [inline xconnect_frontend/xconnect/public/st-xaviers-college.jpg:1]: remove this img

Comment (SelFerns) [reply xconnect_frontend/xconnect/public/st-xaviers-college.jpg:1]: I'll remove it, merge this if there are no more changes required other than that image

**Review by sea-rod — APPROVED** [review]
finally

**Comment by caseynunes** [comment]
> finally

Yayyyy!!!!!!!!!!!!!!

**Pull request merged**